corehq/messaging/scheduling/forms.py

```diff
@@ -3452,6 +3550,15 @@ def clean(self):
             if self.cleaned_data.get('content') != self.CONTENT_EMAIL:
                 raise ValidationError(_("Email case property can only be used with Email content"))
 
+        if self.cleaned_data.get('content') == self.CONTENT_FCM_NOTIFICATION:
+            recipient_types_choices = dict(self.fields['recipient_types'].choices)
+            unsupported_recipient_types = [str(recipient_types_choices[recipient_type])
```

Comment:
won't this be better done with sets?

Reply:
Resolved in 28587859230d0385c494818c53a4b55a9b6eebf2